apps/orams/components/AwardsForm/index.js

```diff
@@ -0,0 +1,2 @@
+import AwardsForm from './AwardsForm'
```

Comment:
why do we need this?

Reply:
I didn't optimize the code. I basically just did as fast as possible the feature, which seems to be the 'ethos' of what we discussed.